vitrina/helpers.py

```diff
@@ -449,41 +458,137 @@ def get_current_domain(request: WSGIRequest, ensure_secure=False) -> str:
     return request.build_absolute_uri(domain)
 
 
+
 def prepare_email_by_identifier(email_identifier, base_template_content, email_title_subject, email_template_keys):
```

Comment:
Do we still need this function? If not, it should be removed.

Reply:
I noticed that people are using this and another function in new functionalities, so I think this function should be left until this task will be merged to devel.